### PR TITLE
Add Ubuntu bionic support

### PR DESCRIPTION
**Pull request by daniel-zullo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -104,6 +104,11 @@ jobs:
         COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb:api-sanity-checker
       before_cache:
         - if [[ -n "$TRAVIS_TAG" ]]; then cp build/pkg/deb/*.deb $CACHE_DIR/; fi
+    - <<: *linux-build
+      env:
+        DIST_VERSION=bionic COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb:api-sanity-checker
```
Comment (bokchan) [inline .travis.yml]: Try to remove the `COMPILER_VERSION` env variable. Pretty sure `gcc 4.9` isn't the default installed on bionic.

Comment (daniel-zullo) [reply .travis.yml]: Yep, that's right. I'll give it a shot.

**Comment by bokchan** [comment]
LGTM

**Review by ben-palmer-sociomantic — APPROVED** [review]
LGTM too although I don't know a heap about this stuff

**Comment by bokchan** [comment]
Also gonna make an internal fork of libdrizzle-redux? Heard DH planned to fork `ocean` ;)

**Comment by daniel-zullo** [comment]
> Also gonna make an internal fork of libdrizzle-redux? Heard DH planned to fork `ocean` ;)

No, I don't think so. Though I guess this library will be abandoned :/

**Pull request merged**